src/index.ts

```diff
@@ -3216,6 +3305,18 @@ export class SunsynkPowerFlowCard extends LitElement {
         return numberValue;
     }
 
+    changeAnimationSpeed(el: string, speedRaw: number) {
+        const speed = speedRaw >= 1 ? this.toNum(speedRaw, 3) : 1;
```

Comment:
Speeds are rounded here to 3 decimals only, so that the UI do not update unnecessarily for small changes